schema/mysql/v57/visibility/versioned/v1.2/add_history_size.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE executions_visibility ADD history_size_bytes BIGINT NOT NULL DEFAULT 0;
```

Comment:
This is different that the schema above. In the schema, you let it be NULL and don't have default value.